docker-compose.prod.yml

```diff
@@ -18,10 +44,33 @@ services:
             - "ASPNETCORE_ENVIRONMENT=Production"
             - "MINITWIT_ENVIRONMENT=docker"
             - "POSTGRES_CONNECTIONSTRING=Host=database;Database=MiniTwit;Username=postgres;Password=test"
-        ports:
-            - "80:80"
-            - "443:443"
         volumes:
             - ./aspnetkeys:/root/.aspnet/DataProtection-Keys
+        labels:
+            # Tell traefik that this container should be exposed to the public
+            - "traefik.enable=true"
+            # Tell traefik which port our container listens on
+            - "traefik.http.services.webapp.loadbalancer.server.port=80"
+
+            # Add http entrypoint (Tell where the service should be available)
+            - "traefik.http.routers.webapp.rule=Host(`minitwit.tk`) || Path(`/{path:.*}`)"
```

Comment:
Could we use a yaml reference here to only have the domain defined in one place?

Reply:
No